docs/project/changelog.md

```diff
@@ -12,33 +12,26 @@ substitutions:
 
 # Change Log
 
-## Unreleased
-
-### Backward incompatible changes
-
-- {{Breaking}} Default working directory(home directory) inside Pyodide virtual
-  file system has been changed from `/` to `/home/pyodide`. To get previous behavior, you can
-  - call `os.chdir("/")` to change working directory
-  - or call {any}`loadPyodide <globalThis.loadPyodide>` with the `homedir="/"` argument
-    {pr}`1936`
+## Version 0.19.0
```

Comment:
```suggestion
## Version 0.19.0

_January 10, 2021_
```

If you want to tag today.

Reply:
I think everything is ready, could you read over the release notes again and check for any last edits?